modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/FormRenderer/FormRenderer.es.js

```diff
@@ -60,6 +60,14 @@ class FormRenderer extends Component {
 	_handleFieldFocused(event) {
 		this.emit('fieldFocused', event);
 	}
+
+	_getDisplayableFn() {
```

Comment:
@natocesarrego , the purpose of this method is to decide if the form should be displayed. I used the "Fn" because it seems to be the pattern when using a function to get an attribute value (git grep "Fn(" "*.js").